projects/plugins/jetpack/modules/sharedaddy/sharing.php

```diff
@@ -743,7 +746,7 @@ function ( $carry, $item ) {
 						<a href="<?php echo esc_url( admin_url( 'site-editor.php?path=%2Fwp_template' ) ); ?>" class="button button-primary">
 							<?php esc_html_e( 'Go to the site editor', 'jetpack' ); ?>
 						</a>
-						<a href="<?php echo esc_url( Redirect::get_url( 'jetpack-support-sharing-block' ) ); ?>" class="button" target="_blank" rel="noopener noreferrer">
+						<a data-target="wpcom-help-center" href="<?php echo esc_url( $link ); ?>" class="button" target="_blank" rel="noopener noreferrer">
```

Comment:
The Jetpack link would break the Help Center. We should make the `data-target` attribute conditional as well.

<img width="414" height="590" alt="Image" src="https://github.com/user-attachments/assets/eedb4769-8e51-4abe-a009-0862067ef33f" />

Reply:
> The Jetpack link would break the Help Center.

Do we load the Help Center in self-hosted Jetpack sites? I don't reproduce the issue when I test it on my site, but maybe it's because I'm missing the scenario in where the Help Center shows up in self-hosted sites 🤔 

https://github.com/user-attachments/assets/0f5fb0ed-eda4-4670-bdb2-324260a5bd1e

Reply:
I didn't make it optional because we don't load the HC library on self-hosted. It shouldn't have any effect on selfhosted.